<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>483</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pbGetMetadata">
       <property name="text">
        <string>Get metadata</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lLat">
         <property name="text">
          <string>LAT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leLat"/>
       </item>
       <item>
        <widget class="QLabel" name="lLng">
         <property name="text">
          <string>LNG</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leLng"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pbGetImageByPidAndZoom">
       <property name="text">
        <string>Get image by pid and zoom</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lPid">
         <property name="text">
          <string>Pid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePid"/>
       </item>
       <item>
        <widget class="QLabel" name="lZoom">
         <property name="text">
          <string>Zoom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leZoom">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lXCoord">
         <property name="text">
          <string>X-coord</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leXCoord">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lYCoord">
         <property name="text">
          <string>Y-coord</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leYCoord">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pbResetAll">
       <property name="text">
        <string>Reset all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbResetNH">
       <property name="text">
        <string>Reset networkhandler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClearCache">
       <property name="text">
        <string>Clear cache</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbInitGrid">
       <property name="text">
        <string>Init grid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="pteLog"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="grid">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
